website/static/js/_anonymousLogTexts.js

```diff
@@ -0,0 +1,126 @@
+    'pointer_forked' : 'A user forked a link to a project',
+    'pointer_removed' : 'A user removed a link to a project',
+    'made_public' : 'A user made a project public',
+    'made_private' : 'A user made a project private',
+    'tag_added' : 'A user tagged a project',
+    'tag_removed' : 'A user removed a tag from a project',
+    'edit_title' : 'A user changed the title of a project',
+    'edit_description' : 'A user edited the description of a project',
+    'updated_fields' : 'A user changed the ${updated_fields} for a project',
+    'external_ids_added' : 'A user created external identifiers on a project',
+    'contributor_added' : 'A user added contributor(s) to a project',
+    'contributor_removed' : 'A user removed contributor(s) from a project',
+    'contributors_reordered' : 'A user reordered contributors for a project',
+    'permissions_update' : 'A user changed permissions for a project',
+    'made_contributor_visible' : 'A user made contributor(s) visible on a project',
+    'made_contributor_invisible' : 'A user made contributor(s) invisible on a project',
+    'wiki_updated' : 'A user updated a wiki page of a project',
+    'wiki_deleted' : 'A user deleted a wiki page of a project',
+    'wiki_renamed' : 'A user renamed a wiki page of a project',
+    'made_wiki_public' : 'A user made the wiki of a project publicly editable',
+    'made_wiki_private' : 'A user made the wiki of a project privately editable',
+    'addon_added' : 'A user added an addon to a project',
+    'addon_removed' : 'A user removed an addon from a project',
+    'addon_file_moved' : 'A user copied a file',
+    'addon_file_copied' : 'A user copied a file',
+    'addon_file_renamed' : 'A user renamed a file',
+    'folder_created' : 'A user created a folder in a project',
+    'file_added' : 'A user added a file to a project',
+    'file_updated' : 'A user updated a file in a project',
+    'file_removed' : 'A user removed a file from a project',
+    'file_restored' : 'A user restored a file from a project',
+    'checked_out':  'A user checked out a file from a project',
+    'checked_in':   'A user checked in a file to a project',
+    'comment_added' : 'A user added a comment to a project',
+    'comment_removed' : 'A user deleted a comment from a project',
+    'comment_updated' : 'A user updated a comment on a project',
+    'embargo_initiated' : 'A user initiated an embargoed registration of a project',
+    'embargo_approved' : 'A user approved an embargoed registration of a project',
+    'embargo_cancelled' : 'A user cancelled an embargoed registration of a project',
+    'embargo_completed' : 'A user completed an embargo of a project',
+    'retraction_initiated' : 'A user A Retraction of a Registration is proposed',
+    'retraction_approved' : 'A user approved retraction of registration of a project',
+    'retraction_cancelled' : 'A user cancelled retraction of registration of a project',
```

Comment:
This and other `retraction_action` logs should have the language updated to replace `retraction` with `withdrawal`.